### PR TITLE
Add React-Native SDK uninstall steps

### PR DESCRIPTION
**Pull request by krystofwoldrich**
Context
- https://github.com/getsentry/sentry-react-native/issues/1139

### CODE REVIEW

File: src/platform-includes/getting-started-install/react-native.mdx [hunk 1]
```diff
@@ -63,3 +63,15 @@ For Android, we hook into Gradle for the source map build process. When you run
 If you have enabled Gradle's `org.gradle.configureondemand` feature, you'll need a clean build, or you'll need to disable this feature to upload the source map on every build.
 
 To disable this feature, set `org.gradle.configureondemand=false` or remove it as its default value is disabled, do this in the `gradle.properties` file.
+
+### Uninstall
```
Comment (marandaneto) [inline src/platform-includes/getting-started-install/react-native.mdx:67]: I'd not put it here, this is the install step and I'd not take their focus away.
TBH I am not sure where to place this, maybe @imatwawana has ideas? I guess no other SDK has it.

Comment (krystofwoldrich) [reply src/platform-includes/getting-started-install/react-native.mdx:67]: I agree I haven't found any other uninstall steps so I wasn't sure where would be the best place.

File: src/platform-includes/getting-started-install/react-native.mdx [hunk 2]
```diff
@@ -63,3 +63,15 @@ For Android, we hook into Gradle for the source map build process. When you run
...
+
+If you want to uninstall the SDK, you can run the following commands:
+
+```bash
+npx @sentry/wizard -i reactNative -p ios android --uninstall
+yarn remove @sentry/react-native
+cd ios && pod install
+```
+
+After that, you can remove the `ios/sentry.properties` and `android/sentry.properties` files. At last, you need to remove all the code using `@sentry/react-native` from your project.
```
Comment (lizokm) [inline src/platform-includes/getting-started-install/react-native.mdx:77]:
```suggestion
To uninstall the SDK, run the following commands:

```bash
npx @sentry/wizard -i reactNative -p ios android --uninstall
yarn remove @sentry/react-native
cd ios && pod install
```

Once that's done, remove the `ios/sentry.properties` and `android/sentry.properties` files. Finally, remove all the code using `@sentry/react-native` from your project.
```

**Review by lizokm — APPROVED** [review]
Hi there! I'm Liza, the new docs manager. Apologies for this being late! I've made some edits. I think it would make more sense for the user to see instruction for how to uninstall in the "next steps" section rather than in the section that's all about how to install it, where it feels out of place. @imatwawana would love your opinion.

**Pull request merged**